docs/static/options.yaml

```diff
@@ -1,8 +1,16 @@
 defaults:
   - architecture: soap_bpnn  # architecture used to train the model
+  - _self_
+
+# Last position of the _self_ this entry defines that default options will be
+# overwritten by this config.
```

Comment:
I don't quite understand. What is self for?

Reply:
This is a hydra thing `_self_` represents the current config. Imagine you have the following default

```yaml
defaults:
  - architecture: soap_bpnn
  - _self_
```

Then the current config overwrites options in `soap_bpnn`. But, if you have the following 

```yaml
defaults:
  - _self_
  - architecture: soap_bpnn
```

`soap_bpnn ` will overwrite the current options. Of course this is not what we want but hydra has this possibility and will throw a waring if you do not define the `_self_` entry in the default list. I will clarify this in the docs!